.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Any benefit of adding the pre-commit hook here instead of `package.json`? Also not sure what line 2 is doing here.

Reply:
This is where husky stores the hooks now: https://typicode.github.io/husky/#/?id=create-a-hook

Line two exists in all these new style husky hooks and calls the husky shell script which is installed at `.husky/_/husky.sh` when `husky install` is run.

Here's an example of the same in the husky repo: https://github.com/typicode/husky/blob/main/.husky/pre-commit

Reply:
There's more info on how this changed from v4-v6 available here: https://typicode.github.io/husky/#/?id=migrate-from-v4-to-v6